controllers/tasks.js

```diff
@@ -489,6 +489,19 @@ const updateStatus = async (req, res) => {
   }
 };
 
+const orphanTasks = async (req, res) => {
+  try {
+    const { lastOrphanTasksFilteration = 0 } = req.body;
+
+    const tasksData = await tasks.updateOrphanTasksStatus(lastOrphanTasksFilteration);
```

Comment:
updateOrphanTasksStatus is returning the count only, right? 
then can we rename the variable `tasksData` to something relatable?

Reply:
Done